asyncapi/asgardeo/service_types.bal

```diff
@@ -21,3 +21,5 @@ public type NotificationService service object {
 };
 
 public type GenericServiceType RegistrationService|UserOperationService|LoginService|NotificationService;
+
+
```

Comment:
Let's keep one line and remove the unnecessary one.

Reply:
Updated as suggested